userguide/content/en/docs/Deployment/_index.md

```diff
@@ -12,20 +10,10 @@ There are multiple possible options for deploying a Hugo site, including Netlify
 
 Depending on your deployment choice you may want to serve your site locally during development to preview content changes. To serve your site locally:
 
-1. Ensure you have an up to date local copy of your site files cloned from your repo. Don't forget to use `--recurse-submodules` or you won't pull down some of the code you need to generate a working site.
-
-    ```
-    git clone --recurse-submodules --depth 1 https://github.com/my/example.git
-    ```
-
-    {{% alert title="Note" color="primary" %}}
-If you've just added the theme as a submodule in a local version of your site and haven't committed it to a repo yet,  you must get local copies of the theme's own submodules before serving your site.
-
-    git submodule update --init --recursive
-    {{% /alert %}}
+1. Ensure you have an up to date local copy of your site files cloned from your repo.
```

Comment:
I simplified this because, IMHO, we shouldn't be telling folks how to clone their own repos -- in particular because we don't know the flags that they require.